src/Forms/GridFieldRefreshButton.php

```diff
@@ -164,8 +182,37 @@ public function hasActiveJob()
     public function handleRefresh()
     {
         if (!$this->hasActiveJob()) {
-            $injector = Injector::inst();
-            $injector->get(QueuedJobService::class)->queueJob($injector->create(CheckForUpdatesJob::class));
+            // Queue the job in the immediate queue
```

Comment:
`if ($this->hasActiveJob()) { return; }` might look slightly cleaner - thoughts?

Reply:
👍Agreed. Return early